contrib/kind.sh

```diff
@@ -369,8 +388,22 @@ run_kubectl apply -f ovnkube-master.yaml
 run_kubectl apply -f ovnkube-node.yaml
 popd
 
+pushd ${tmp_admission_dir}
+run_kubectl -n ovn-kubernetes create secret tls ovnkube-admission-controller-secret \
+    --cert "admission-server-tls.crt" \
+    --key "admission-server-tls.key"
+popd 
+rm -rf ${tmp_admission_dir}
+
 # Delete kube-proxy
 run_kubectl -n kube-system delete ds kube-proxy
+# Create the admission deployment after having deleted kube-proxy so that it does not create 
+# the iptables REJECT rules before being deleted
```

Comment:
maybe you can check that kube-proxy does not exist with an active loop

Reply:
Good point. It seems `kubectl delete --wait=true` should do the same though, I will use that.